pkg/tests/tasks/security/authorization/ext_auth_test.go

```diff
@@ -129,7 +195,27 @@ spec:
     - operation:
        paths: ["/headers"]
 `
-
+	ExternalRouteGrpc = `
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: ext-authz
+  namespace: foo
+spec:
+  selector:
+    matchLabels:
+      app: httpbin
+  action: CUSTOM
+  provider:
+    # The provider name must match the extension provider defined in the mesh config.
+    # You can also replace this with sample-ext-authz-http to test the other external authorizer definition.
+    name: sample-ext-authz-grpc
+  rules:
+  # The rules specify when to trigger the external authorizer.
+  - to:
+    - operation:
+       paths: ["/headers"]
```

Comment:
This indentation is still invalid.
```suggestion
        paths: ["/headers"]
```

Reply:
No, under "e" (`paths` is a child of `operation`, so it must be indented by two spaces relative to the parent (`operation`)).